.github/workflows/buildkite.yml

```diff
@@ -0,0 +1,29 @@
+name: Approve PR for running Buildkite pipelines
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  add_ci_label:
+    name: "Add CI label to PR"
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/ci') && github.event.comment.author_association == 'MEMBER'
+    runs-on: ubuntu-latest
```

Comment:
I believe this will trigger even I write `/ci` by mistake inside a larger conversation.

What if we replace it with [startsWith](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#startswith) ? This way only a comment literally with `/ci` in its body will trigger this

Reply:
OK switched to startsWith instead of contains.